### PR TITLE
adding BHNT #61

### PR DESCRIPTION
**Pull request by Wikinaut**
+ as a starting point

### CODE REVIEW

**Comment by vog** [comment]
Thanks! However, the Meetup URL and title should be adjusted to BHNT#61.

**Comment by vog** [comment]
Meetup URL: https://www.meetup.com/de-DE/Berlin-Hack-and-Tell/events/251010410/

Title: Revolutionary May Hacks

**Comment by Wikinaut** [comment]
@vog ty for spotting this. What's about the date and time, I have fixed them, too. (pls. let me know whether that's fine)

**Comment by vog** [comment]
Whoops, that was indeed "fixing too much".

The time 18:30 was correct (that's when the event starts, even though presentations usually start around 19:00).

The time component of the "date" was also correct - its value "`13:37:42 +0100`" has technical reasons.

**Comment by Wikinaut** [comment]
done. Wo habt Ihr diese "Werte" festgelegt, so dass newbies wissen, was tun (oder nicht tun sollen)?

**Comment by Wikinaut** [comment]
@vog 
skeleton usually before the event: ok.

#### tl;dr:

I would like to join your team to help in preparing (with a pad) and documenting BHNT (using your current workflow).

#### long version

CCC-Berlin usually uses a simple pad (with a non-public url) i) in the preparation phase and ii) then during the event; this is very handy and nice, as it can be fixed during the BHNT by attendees and speakers, and is usually up-to-date and with all the needed information when the HNT ends.

After the event, the relevant pad content can be casted into a git repo as in your current workflow. 

Ligi told me, that he wanted to have a kind of "surprising effects", so that attendees do not know the exact schedule for the event in advance (I can partly agree here).

I suggest this for BHNT, too.
